src/components/main/Main.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 
-import Content from '../content';
-import Tasklist from '../Tasklist';
-
 import './main.scss';
 import { checkUser } from "../../services";
 import Login from "../../pages/login";
+import Dashboard from "../../pages/Dashboard";
```

Comment:
1) use single quotes
2) there is no here `Dashboard` folder
https://github.com/ex3emxl/To-Do/tree/feat/routing/src/pages/dashboard